modules/universal/scalardl/main.tf

```
@@ -173,16 +177,33 @@ resource "null_resource" "schema_loader_image_load" {
   }
 }
 
+resource "null_resource" "scalardl_container_env_file_push" {
+  count = var.provision_count > 0 ? 1 : 0
```

Isn't this necessary for all nodes?
```suggestion
  count = var.provision_count
```